src/Codeception/Coverage/Filter.php

```diff
@@ -73,7 +73,7 @@ public function whiteList($config)
         if (isset($coverage['whitelist']['include'])) {
             foreach ($coverage['whitelist']['include'] as $fileOrDir) {
                 $finder = strpos($fileOrDir, '*') === false
-                    ? [$fileOrDir]
+                    ? [Configuration::projectDir() . DIRECTORY_SEPARATOR . $fileOrDir]
```

Comment:
I think that this code creates an incorrect path when codeception.yml is included. #3433 
@loren-osborn what do you think about it?